modules/cassandra/src/test/java/org/testcontainers/containers/CassandraContainerTest.java

```diff
@@ -106,8 +106,23 @@ public void testCassandraGetCluster() {
         }
     }
 
+    @Test
+    public void testAuthenticationSupportWithWaitStrategyAndInitScript() {
```

Comment:
I don't think we need `WithWaitStrategyAndInitScript` part here, it should be enough to verify that, when authentication is enabled, the container starts correctly and can be queried. WDYT?

Reply:
As the init script and wait strategy both connect to Cassandra cluster, they need to fetch the proper Session using proper credentials. Thus I think we should definitely have a test case by using both. But I agree there should also be another test that doesn't take anything else and makes sure that the connection to the cluster can be made. I will add some the same.

Reply:
Added in Latest Commit. Please re-review.

Reply:
@KraTos5589 

1. please, do not mark conversations that were not started by you as resolved, it is up to the reviewer
2. wait strategy is always applied, no need to test it separately, `testAuthenticationSupport` is enough
3. once again, there is no need for this (`testAuthenticationSupportWithWaitStrategyAndInitScript`) test. Things are tested separately